corehq/apps/reports/standard/users/reports.py

```diff
@@ -22,6 +26,29 @@
 from corehq.util.timezones.conversions import ServerTime
 
 
+def get_primary_properties(domain):
```

Comment:
nit: `get_user_history_primary_properties` since this for use for just the User History Report?

Reply:
oops, I lost track of where this function was, I'd thought it was in a users file. Updated in 62a20d3fbfd to make it a class method of the report, which namespaces it.